NOTICE

```diff
@@ -0,0 +1,16 @@
+This NOTICE pertains to some code snippets shown in [README.md](/README.md).
+
+Copyright 2021 DFINITY Stiftung
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+[README.md](/README.md) contains derivative works licensed as MIT: https://github.com/demergent-labs/azle/blob/main/LICENSE
```

Comment:
I'd highly recommend turning on the vscode setting "Files: Insert Final Newlines". Prettier can't take care of this for unknown file types. Plus, since it's the POSIX standard it should have them. This way you just never have to think about it again.

Additionally I'd recommend enabling the "Files: Trim Final Newlines" and "Files: Trim Trailing Whitespace" settings. Again, they do what you're really wanting and you just don't have to think about it.